src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -1752,6 +1818,8 @@ public DatabaseStatusReport GetPreviousDatabaseReport(string node)
 
                 return databaseReport;
             }
+
+            public static implicit operator MergedDatabaseObservationState(DatabaseObservationState state) => new MergedDatabaseObservationState(state.RawDatabase);
```

Comment:
Why use implicit operator overloading?

Reply:
Simply so I will not to have modify any existing test that use `DatabaseObservationState`

Reply:
How many of those do we have? I'm nervous about implicit state like that

Reply:
the usages are only in tests and seems we have 5 of them